### PR TITLE
[FEATURE] Add new feature to set the table name

### PR DESCRIPTION
**Pull request by franzholz**
Similar to mapToTable, however no table needs to exist.

### CODE REVIEW

**Comment by simonschaufi** [comment]
- Tests missing
- This file must be generated in order for it to work: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87623-ReplaceConfigpersistenceclassesTyposcriptConfiguration.html

**Review by simonschaufi — CHANGES_REQUESTED** [review]
This is still far away from being reviewed.

- Tests missing
- This file must be generated in order for it to work: https://docs.typo3.org/c/typo3/cms-core/master/en-us/Changelog/10.0/Breaking-87623-ReplaceConfigpersistenceclassesTyposcriptConfiguration.html with the  `tableName` mapping

File: Resources/Private/Language/locallang.xlf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" standalone="yes" ?>
 <xliff version="1.0">
-	<file source-language="en" datatype="plaintext" original="EXT:extension_builder/Resources/Private/Language/locallang.xlf" date="2012-02-29T20:52:35Z"
+<file source-language="en" datatype="plaintext" original="EXT:extension_builder/Resources/Private/Language/locallang.xlf" date="2012-02-29T20:52:35Z"
```
Comment (simonschaufi) [inline Resources/Private/Language/locallang.xlf:3]: please revert this change.

Comment (franzholz) [reply Resources/Private/Language/locallang.xlf]: undone

**Comment by franzholz** [comment]
In the meantime, 2 days ago, the tests have changed.
So I can hardly add the tests to this patch which is based on an older version of the code.

**Comment by simonschaufi** [comment]
Just rebase on master and force push.

**Comment by PKuhlmay** [comment]
I will close this, as the GUI had changed completely and the old JS is not used anymore. If you still need this for v12, then please create an issue for this.

**Pull request closed without merging**